app/src/main/java/com/android/sample/ui/filter/FilterPage.kt

```diff
@@ -184,6 +185,23 @@ fun FilterBox(
       }
 }
 
+/**
+ * Composable function to reformat the time.
+ *
+ * @param time The time to reformat in minutes.
+ */
+@Composable
+fun reformatTime(time: Float): String {
```

Comment:
The changes in this file are solid and well-implemented. There’s only one thing to refactor: replacing the hardcoded `60` in `reformatTime` with the `MINUTES_PER_HOUR` constant from `C.kt` for better maintainability.

Reply:
Nice catch! I ll do it right away! 😮